.travis.yml

```diff
@@ -0,0 +1,13 @@
+env:
+  global:
+  - FLASK_APP=todo_app/app
+  - FLASK_ENV=development
+  - secure: "QzzxBZuczObJ5X6RVRCOQ4JaSdcoo/0krxLMoRTm5+bNSko1eoYHp+PpSDuENhuvOPfYc6nquYcK4EG5juGtaXoLeYeZ5z35j85SL+26xbDAh9JMZhb1j0Ai5MdVWXkwKfsWFb56GTgsFOykmWqvhYavWj57SRkMoHxYeIkneHtYyU2mlOREPFGBM97SlTUUEkGuQ4QNN8VOOgKxopM8x7UD7SrAwVPibrHxoWlIBvC1gEcBWlfmiroE9xiFpFIg3N6Atrf28ajnwaHU6pUcZrXpskTUa02NLE6RbXUtvlV8xe5+SK+RcZkyq4YGpnyD9VONOgH2W35XsIhggm15mjFi53tY4Nnx3QwyopMcHd7p1PRx/rlcTFUZNpkRLSM0pjCE9imKEe98JK90pAYmaUKgQt0lfDF2vuxUavbFIZxJLQRasEbzD778fQoMibDhd4FmyBqsyfR389L/lH5bj7yejtRnfj4A7MQVvX/i8AE18Desvlqmuyug/vhFI4eDLIp6YdQvC4n3MLupifq96vCFcr4j9KReO+nPyMk7lhX9oGlP7bKJJOnqbJRLOUs1PsmJlHUCoxKlV83Fxj81Lgp4PpqiGmlqOkkkq++pzVDhb18gwZSGabto+pOeQzXDWsKz/ay1CG8AK7m9fXDp2IPAfUEXaCYyCa9/el1Ewtg="
+  - secure: "off3vGvFxgajHVk3XYEuGlym5AN/bebl88lA49I3n8Z8IL5VxOtB4wxRmfx0JH5QVZopm4wzTR1JN86SjWBuCWnMNhK03TME09rvnWM6C29LKVijTU6GC0L+/gtPHtujt+Mcs+VHsBEJacHvVJVkAHHz+F8vUCqZmf23p/GG5wWAdSI/jS09zbEOS9KLz5rmtDt2UOuhMxfzvvw8Kqbo7WwoGXcuwqjSiZPa5Esuf2dYkPyrrPLUgaK/8byL2CDvUTA5NIcySNLkLhJ/ccc3zskbEj8DF95Zu/3MVhJmgUEn6U93V62pK+aFpNWYpVeHdi22yTDIr/mXuC7afXdJJWSAiZhTAK9dS6BvrAHzEoaiOfNBvFxBpZZI+04tjoLzWQqUNtUpcGTeTj6svdaQyT9eIUA5+ZCHJqujgFxYj7MVoNjzz4ehtSF/vndfpHV+Y8z9CcASr4J10BjJNHRvrrFV+v2JJEDpSlPA0F0cSWzfLdy4Scu2TuPYu2pqyluTFmQ3hclYfd8D7wRLJ+nedb0lo2xybxCubKfleAGAw7zALbf/rwQf9BzF7xqkm541NsFAYr8mPTXPnjfBvsPGEXigahB3xzLQjLpoLYEdCx78OxqkmuilvDhFsCYJC+G42Hc6vb16YT89Z0GjVlwCNF7yjS2dBlHFSCh6SovCSzA="
+  - secure: "jp9Q3Jrc/J4S92GxXR0PYej0w3dqH7yTKTED5Un6KWGq4JRzPN4snhO0ynYjRKeR3hzeQ8zK+kJUFnFBedqYXgYNWhZ2oEGX3j6OQyC9KAwH2Pjagjp5WHjJhsRkCZwxwRpwjZ02mnHrbKsl78+41Gc28/BjkyhZxaYwYiiuE5bYLzxGtcar8Cnuu2l4Ls4DunrDvQ2rS9Ajfkn6wUm7LP4n6hilLOGzVXk78nol6DmpudQCmuNVJ1tgKkC7WoQ1InMLwd3BKiybR1vtMbpv4W02GS7AiGn/62j7NesrX7T/5khDW1JlcB83+cAWvzKpXoAi+0Ufk1nnFooqa4bfWGRo7iheIb7Igy7Z0cI+M0Fds2Th1yiUYDTJbqvcCSfn52+KKYbgRFa1P5HqdNkgKpsKoeW7T4vfv9T6OLfL5Ezswvzs5X1l5LG/ee4hDo23pSFuWf/oDlQd6AWYXs/h+WZF43PlL3q1wp4uyKUjwVD3XyPI6D9XF6cmFNYQSHymYZsiCX+Uko11K1oI2PmcisWP0Ep0Mt1MyPGPYy8kdK3msCCGwPune5GRLucp2ILiAv5mv2sK4gd4OqSYgIyT2uu8+7fIx7tg3g4fTv6AOD4M40ZAGQhRYwvD+IAfEHlXk4sw2vszJpcBXFcLciw13XC2Jkp4ApF68EXwL4IZpwo="
+  - secure: "B0WQnN6dImOotOTMBg3dGxCsjAo8EA+14KOBkk3tTU9PM1+mCiyP4BCf7OYAVh/0wKDTkPOT4+lOGinoy7kScPwogt2hAZqE3HpvLZfJK+oNZK3laB3m0lAe4ZokVpt78xdppL6T3WPcgC5w0yUN7sgSHe/7w6rDUJOR23/TMNWDppYlVSjb+EBZiHluQbWfXIY6DR2MYGAsVeWceW9VCOEkS45T8IqTPKe2GDdZQ5DpekPwgvH999VioY8kCE+TjOilHBV8jnKjqsIbx/vUy6CuMhRrsm7ss7KmxtynZbFef+K6royzQLATSkjT5SengPLgtAcz343JX4U2qhxT2dDbkpX9Bz8lqk+UkEeEVqbzTM0Q+bBMbvwsUggCWiZcX/zh/PEE9GtOWjPLfeaTHi6sG22TrML6X+8wRO5OaMgjFnOlH7uj3gKIuxdRkNbgQZ7poP4/sLqO12d0KMbKDavLuR7hOs8xSydKVroK82ig7dx6iUSIw18h4XkQNSZ80X91Myi2SMeWjZ/SfpHuE8+wrBMB+CaXB0p1RxeTEcr/7mQ9G6U30eMkKJAlnl7lpb1smI8ByUCtZu1q23tuKjKkVN4wHSNERVXXTzyXoku2cq14p79+xOF+T1Ogb9YdAQtGLrXnOPwMHlhI2xWZMrTFgogk79Y0Y2qR2aun/z0="
+script:
+- docker build --target test --tag my-test-image .
+- docker run my-test-image todo_app/test_ViewModel.py
```

Comment:
You're not running the integration test here, is there a reason? To make this more easily extensible, you might like to put the unit/integration tests in a `tests` folder, such that we can run those (and any new non-e2e tests) with a single docker command, like
`docker run my-test-image todo_app/tests`

(Running your integration test failed for me locally just because there's a mismatch of environment variables; i.e. you might want to change your `.env.test` file to have the test board id; `TRELLO_BOARD_ID=6006195ae74f187322fda449`)